tuxemon/db.py

```diff
@@ -233,6 +233,13 @@ def valid_level(cls, v):
             raise ValueError(f"invalid level learned: {v}")
         return v
 
+    @validator("level_learned")
+    def is_multiple(cls, v):
+        if v > 0:
+            if has.multiple(v, 2):
+                return v
+            raise ValueError(f"no resource exists in db: {v}")
```

Comment:
this error message is misleading

Reply:
I'm going to remove it, I used it for maintaining order